docs/content/how-to/disconnected/known-issues.md

```diff
@@ -189,3 +189,56 @@ for isName in ${IMAGESTREAMS}; do
   oc patch -n openshift ${isName} --type json -p '[{"op": "replace", "path": "/spec/tags/0/importPolicy/insecure", "value": true}]'
 done
 ```
+
+## IDMS/ICSP with only root registry are not propagated to the registry-override flag under ignition server
+
+See: [OCPBUGS-33951](https://issues.redhat.com/browse/OCPBUGS-33951)
+
+**Symptoms:**
+Even having the ICSP/IDMS well set in the Management cluster and working fine in that side, the HostedControlPlane deployment is not capable of extract the metadata release fromthe OCP payload images. Something like this log entry will show up in the ControlPlaneOperator or HypershiftOperator pod.
+
+```
+failed to lookup release image: failed to extract release metadata: failed to get repo setup: failed to create repository client for https://registry.sample.net: Get "https://registry.sample.net/v2/": tls: failed to verify certificate: x509: certificate signed by unknown authority
+```
+
+**Root Cause**:
+Once the ICSP/IDMS are created in the Management cluster and they are only using in the "sources" side a root registry instead of pointing a registry namespace, the image-overrides are not properly propagated through all the deployments in the HostedControlPlane namespace. This is how looks like:
```

Comment:
Is this saying the API input is legit but we don't propagate it properly? can we fix that?

Reply:
Yes, I have a bug opened but it's set to the next sprint so maybe will not get in to 4.16.0 relase cut

Reply:
can you please share link?

Reply:
https://issues.redhat.com/browse/OCPBUGS-33951